src/together/resources/finetune.py

```diff
@@ -397,6 +464,15 @@ def download(
             FinetuneDownloadResult: Object containing downloaded model metadata
         """
 
+        if re.match(r"^ft-[\dabcde-]+\:\d+$", id) is not None:
```

Comment:
I might be misreading the regular expression, but why do you have the character set `abcde` for parsing job IDs? I believe UUIDs can contain at least `f` :)

Reply:
F